README.md

```diff
@@ -1118,20 +1118,21 @@ In either case:
        an attribute when `self` is an ActiveRecord model: `self.guest = user`.
     3. Referencing the current instance's class: `self.class`.
 
-* <a name="freeze-constants"></a>When defining arrays and hashes meant to be
-    constants, make sure to call `freeze` on them. ([More on this][ruby-freeze].)
-    <sup>[[link](#freeze-constants)]</sup>
+* <a name="freeze-constants"></a>When defining strings, arrays or hashes meant to 
```

Comment:
Good point. Should we make the rule generic then to include all mutable types? Maybe say something like:

When defining an object of any mutable type meant to be a constant, make sure to call `freeze` on it. Common examples are strings, arrays, and hashes. 

(Rubocop only seems to allow enforcement of these three btw).